views/calendar/_form.php

```diff
@@ -4,23 +4,26 @@
 use humhub\modules\ui\form\widgets\ActiveForm;
 use humhub\modules\ui\form\widgets\ColorPicker;
 use humhub\widgets\Button;
+use humhub\widgets\Pjax;
 
 /* @var $this yii\web\View */
 /* @var $model \humhub\modules\external_calendar\models\ExternalCalendar */
 /* @var $contentContainer \humhub\modules\content\models\ContentContainer */
 /* @var $form yii\widgets\ActiveForm */
 
-if ($model->color == null && isset($contentContainer->color)) {
+if (!isset($model->color) && isset($contentContainer->color)) {
     $model->color = $contentContainer->color;
-} elseif ($model->color == null) {
+} elseif (!isset($model->color)) {
     $model->color = '#d1d1d1';
 }
 
 \humhub\modules\external_calendar\assets\Assets::register($this);
 ?>
 <div class="calendar-extension-calendar-form">
+    <?php Pjax::begin(); ?>
```

Comment:
Why the Pjax?

Reply:
I have removed it. I  beilive it is not used there.  I have merged with latest master branch, so please check if it is okay, thank you!